pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/util/ValueReader.java

```diff
@@ -20,6 +20,9 @@
 
 import java.io.Closeable;
 import java.math.BigDecimal;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import org.apache.pinot.segment.spi.memory.PinotDataBuffer;
```

Comment:
there might be a better place for the shared logic than a static on this interface, which would keep `PinotDataBuffer` out of this interface. I'm open to suggestions.